openassessment/xblock/test/test_staff.py

```diff
@@ -253,3 +269,107 @@ def test_assessment_error(self, xblock):
             resp = self.request(xblock, 'staff_assess', json.dumps(STAFF_GOOD_ASSESSMENT), response_format='json')
             self.assertFalse(resp['success'])
             self.assertIn('msg', resp)
+
+
+class TestStaffTeamAssessment(StaffAssessmentTestBase):
+    """ Test Staff Team Assessment Workflow"""
+
+    @scenario('data/team_submission.xml', user_id='Bob')
+    def test_staff_assess_handler(self, xblock):
+
+        submission = self._setup_xblock_and_create_team_submission(xblock)
+        submission["uuid"] = str(submission["submission_uuids"][0])
+
+        self.submit_staff_assessment(xblock, submission, assessment=TEAM_GOOD_ASSESSMENT)
+
+        assessment = teams_api.get_latest_staff_assessment(submission['team_submission_uuid'])
+        self.assertEqual(assessment['points_earned'], 2)
```

Comment:
This feels like too many asserts, perhaps a dictionary equal assert or a helper function to do this?